tests/test_snakemake.py

```diff
@@ -11,10 +11,15 @@
 import os
 import pytest
 import sys
+from pathlib import Path
 
 from reana_commons.snakemake import snakemake_load
 
 
+@pytest.mark.xfail(
+    sys.version_info >= (3, 11),
+    reason="Test expted to fail for python versions 3.11 and above as we currently return only empty dictionary in snakemake_load function for python >= 3.11. Development is blocked until we move validation to the server side.",
```

Comment:
Typo: "expected" to fail

Reply:
You can also remove the last part "Development is blocked until we move validation to the server side" to make the reason shorter.  We'll revamp this anyway as part of the server-side validation sprint.